src/main/java/org/apache/commons/lang3/text/StrBuilder.java

```diff
@@ -3045,14 +3045,14 @@ public void write(final int c) {
 
         /** {@inheritDoc} */
         @Override
-        public void write(final char[] cbuf) {
-            StrBuilder.this.append(cbuf);
+        public void write(final char[] buffer) {
```

Comment:
-1: Better to keep the name matching the overridden method from java.io.Writer, it will be less of a surprise if you know the Writer API.